<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Steganography</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(38, 31, 31);
font: 75 italic 11pt &quot;Noto Serif CJK JP&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="EncDecTab">
      <property name="styleSheet">
       <string notr="true">color:white;
QTabWidget::tab-bar{background-color: green;}
QTabBar::tab{
background-color: green;
}


</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="EncTab">
       <property name="styleSheet">
        <string notr="true">border:1px solid;
border-color: rgb(252, 233, 79);
QWidget::selected {
  background-color: green;
 }
</string>
       </property>
       <attribute name="title">
        <string>Encrypt</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="encInputLayout">
          <item>
           <widget class="QPlainTextEdit" name="encInputFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>10000000</height>
             </size>
            </property>
            <property name="plainText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="encOpenFile">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover {
    background-color: rgb(138, 226, 52);
color:black;
};</string>
            </property>
            <property name="text">
             <string>Open File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="encBackImage">
          <item>
           <widget class="QFrame" name="encBackImageFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="encImageLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">border:none</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="encOpenImageButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover {
    background-color: rgb(138, 226, 52);
color:black;
};</string>
            </property>
            <property name="text">
             <string>Open Image in which to Encrypt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="encOutputLayout">
          <item>
           <widget class="QPushButton" name="encButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover {
    background-color: rgb(138, 226, 52);
color:black;
};</string>
            </property>
            <property name="text">
             <string>Encrypt Text In Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="encOutputFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="encOutputImageLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">border:none</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="encSaveButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover {
    background-color: rgb(138, 226, 52);
color:black;
};</string>
            </property>
            <property name="text">
             <string>Save Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DecTab">
       <property name="styleSheet">
        <string notr="true">border:1px solid;
border-color: rgb(252, 233, 79);
</string>
       </property>
       <attribute name="title">
        <string>Decrypt</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="decInputLayout">
          <item>
           <layout class="QVBoxLayout" name="decImageLayout">
            <item>
             <widget class="QFrame" name="decInputImageFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="decInputImageLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:none</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="decOpenImageButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover {
    background-color: rgb(138, 226, 52);
color:black;
};</string>
            </property>
            <property name="text">
             <string>Open Image to Decrypt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="decOutputLayout">
          <item>
           <widget class="QPushButton" name="decButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover {
    background-color: rgb(138, 226, 52);
color:black;
};</string>
            </property>
            <property name="text">
             <string>Decrypt Text from Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="decOutput">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decSaveButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::hover {
    background-color: rgb(138, 226, 52);
color:black;
};</string>
            </property>
            <property name="text">
             <string>Save File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
